File: Backend.Net/BusinessDomain/Wagon.cs
namespace BusinessDomain
{
    public sealed class Wagon : IDomainObject
    {
        public string WagonNumber { get; set; }

        public string Info { get; set; }

        public string TotalWeight { get; set; }

        public string SelfWeight { get; set; }

        public int NumberOfAxles { get; set; }

        public int MaxVelocity { get; set; }

        public Cargo Cargo { get; set; }
    }
}

File: Backend.Net/BusinessDomain/typescript/Wagon.ts


 

    export interface Wagon { 
        wagonNumber: string;
        info: string;
        totalWeight: string;
        cargo: Cargo;
        reservation: ShuttleReservationListItem;
        reservationTrainNumber: string;
        reservationSendDate: string;
        reservationReceiveDate: string;
        wagonStatus: WagonStatus[];
    
}